plane/source/docs/quadplane-frame-setup.rst

```diff
@@ -21,24 +21,34 @@ Frame Types and Classes
 -----------------------
 
 To use a different frame type you can set :ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` and
-:ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>`. :ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` can be:
+:ref:`Q_FRAME_TYPE<Q_FRAME_TYPE>`. 
+
+Frame Class
+~~~~~~~~~~~
+
+:ref:`Q_FRAME_CLASS<Q_FRAME_CLASS>` designates the number of motors used, and can be:
 
 -  1 for quad
 -  2 for hexa
 -  3 for octa
 -  4 for octaquad
--  5 for Y6
--  7 for Tri
--  10 for Tailsitter
+-  5 for Y6 (ignores all following frame classes except 10 and 11, all others result in Y6A configuration)
+-  7 for tri (ignores all following frame classes except 6 for reversed tricopter)
```

Comment:
shouldn't "ignores all following frame classes" be "ignores frame type"?
And class 10 completely ignores frame type, I think.